diff -Naur FTL/dnsmasq/config.h FTL-snap/dnsmasq/config.h
--- dnsmasq/config.h	2018-11-06 22:57:03.686524083 +0000
+++ dnsmasq/config.h	2018-11-06 23:22:48.437456951 +0000
@@ -214,7 +214,7 @@
 #   if defined(__ANDROID__)
 #      define RUNFILE "/data/dnsmasq.pid"
 #    else
-#      define RUNFILE "/var/run/dnsmasq.pid"
+#      define RUNFILE "/var/snap/pihole/common/var/run/dnsmasq.pid"
 #    endif
 #endif

diff -Naur FTL/socket_client.c FTL-snap/socket_client.c
--- socket_client.c	2018-11-06 22:57:03.694524037 +0000
+++ socket_client.c	2018-11-06 23:22:54.681420214 +0000
@@ -40,7 +40,7 @@
 	address.sun_family = AF_LOCAL;

 	char *command = ">stats";
-	strcpy(address.sun_path,"/var/run/pihole/FTL.sock");
+	strcpy(address.sun_path,"/var/snap/pihole/common/var/run/pihole/FTL.sock");

 	int i;
 	for(i = 1; i < argc; i++) {
